packages/xrpl/test/models/utils.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-deprecated -- using deprecated setTransactionFlagsToNumbers to ensure no breaking changes */
 /* eslint-disable no-bitwise -- flags require bitwise operations */
```

Comment:
can you add unit tests for `convertTxFlagsToNumber` method, at least along similar lines as line 51 in this file? Have you added the tests elsewhere?